src/test/java/com/avaya/letscode/TestingCart.java

```diff
@@ -119,6 +119,7 @@ public void testModifyQuantityOfProductInCart(){
         testCart.AddItem(patanjaliSoap, 5,"FIRST2OFF50");
         Assert.assertEquals(6,testCart.getQuantity(doveSoap));
         Assert.assertEquals(5,testCart.getQuantity(patanjaliSoap));
+        Assert.assertEquals(11,testCart.getQuantity());
```

Comment:
This was the missing assert which I have pushed into this branch. This is a failing test now.